src/features/auth/components/signin/Login.tsx

```diff
@@ -45,12 +48,45 @@ export const Login: FC = () => {
     };
   };
 
+  const form = useForm<z.infer<typeof login>>({
```

Comment:
do type inference inside schema.ts file, use the type here, as the above one
`useForm<Type>(...)`